### ISSUE TITLE
Looks like the ping server is down

### ISSUE THREAD

**Issue by benzon** [Itokoyamato/TokoVOIP_TS3#259]
Looks like the heartbeat/ping routing server call it what you want is down - would it somehow be possible for you to release the code for it, so we can host that part our self?

**Comment by akassl-lab** [comment]
I can confirm that the server responsible for the ws_server ping is no longer responding and it's causing issues for servers that is using TokoVOIP. Players are no longer able to connect to the TokoVOIP server

**Comment by badcodestv** [comment]
+1 

Not responding and I'd love to see us have the ability to host the code on our side.

**Comment by The-Gubna** [comment]
Yes, being able to host on our end would be wonderful. Hopefully your team gets this situated.

**Comment by akassl-lab** [comment]
Here's the main reason -> https://twitter.com/olesovhcom/status/1369478732247932929

This is out of the control from Itokyoyamato. This building is the main provider for the TokoVOIP master

**Comment by Eric1212** [comment]
The fact it's does a heartbeat ping is kind of streange and feel useless outside of non-official use / botnet.

**Comment by akassl-lab** [comment]
> The fact it's does a heartbeat ping is kind of strange and feel useless outside of non-official use / botnet.

The heartbeat is for the handshake with the master server. Even if you remove the heartbeat, you still cannot connect to the master server.

There used to be a way around it by hosting the master locally on the client side but since some restrictions within FiveM, you cannot anymore. The only way is to remake the master and host it yourself or if Itokoyamato releases the master source so we can host it ourselves.

**Comment by phiomet** [comment]
So we have to wait and can do nothing if we didn´t want to change the voice solution?

**Comment by daddyglock** [comment]
It's going to be a while, if not ages for them to go through back ups... 
![image](https://user-images.githubusercontent.com/56209548/110601999-72b08c00-817d-11eb-9b73-9d7127a5e546.png)

**Comment by benzon** [comment]
Ye i would more then gladly try and help here, but the handshake part is not public so it's gonna be a bit hard to do anything.

**Comment by The-Gubna** [comment]
> My ws_server says "Heartbeat sent" , Working again?

Yeah, they seem to have it working again, quick turn around for such a disaster

**Comment by akassl-lab** [comment]
This issue has been resolved but let's keep this thread open to use this as an example on why we need to host our own master.
